theano/misc/pycuda_example.py

```diff
@@ -384,7 +381,7 @@ def local_pycuda_gpu_elemwise(node):
     """
     if isinstance(node.op, GpuElemwise):
         if (not any([any(i.type.broadcastable) for i in node.inputs]) and
-            all([i.ndim <= 2 for i in node.inputs])):
+                all([i.ndim <= 2 for i in node.inputs])):
             new_op = PycudaElemwiseSourceModuleOp(node.op.scalar_op,
                                                   node.op.inplace_pattern)(
                                                       *node.inputs)
```

Comment:
Line 401 below uses PycudaElemwiseKernelOp, but its definition is commented out.
Should we comment out that function as well?